src/services/permissions/study.ts

```diff
@@ -15,7 +19,10 @@ export const canReadStudy = async (
     return false
   }
 
-  if (study.isPublic && (await checkOrganization(user.organizationId, study.organizationId))) {
+  if (
+    (study.isPublic || isAdministratorOnStudy(user, study)) &&
+    (await checkOrganization(user.organizationId, study.organizationId))
+  ) {
```

Comment:
C'est pas plutot (même si en vrai ici c'est pas faux mais j'ai l'impression que ca fait des checks en doublon) : 
- IsAdminOnStudy 
- OU (studi is Public ET check orga)